ayushi/ayushi_postgresql_assignment/src/controller/class.ts

```diff
@@ -0,0 +1,51 @@
+import { Context } from "vm";
+
+const {getClassList,addStudentToClassList, getStudentsByClass} = require('../services/class');
+
+interface student_class{
+  student_id: number;
+	class_id: number;
+}
+
+export async function getclassList(ctx: Context){
+  try {
+    const classList = await getClassList();
+    ctx.response.type = 'application/json';
+    ctx.response.status = 200;
+    ctx.response.body = JSON.stringify(classList);
+  } catch (e) {
+    ctx.response.type = 'text/html';
+    ctx.response.status = 500;
+    ctx.response.body = 'data not fetched';
```

Comment:
Can we use "internal server error" ?
Using json for response would help ?
Can we maintain response.type across application?